src/current/cockroachcloud/connect-to-a-serverless-cluster.md

```diff
@@ -33,7 +33,7 @@ AWS PrivateLink for CockroachDB {{ site.data.products.serverless }} is in **[lim
 
 Amazon Web Services (AWS) PrivateLink support allows customers to establish SQL access to their clusters entirely through private AWS infrastructure, without exposure to the public internet, affording enhanced security and performance.
 
-AWS PrivateLink is available only for multiregion CockroachDB {{ site.data.products.serverless }} clusters deployed on AWS.
+AWS PrivateLink is available only for multi-region CockroachDB {{ site.data.products.serverless }} clusters deployed on AWS.
 
```

Comment:
@gemma-shay PrivateLink is still Limited Access until Feb - shouldn't this be a "contact sales/support if you want PrivateLink for Serverless"?